tests/integrations/tests/test_google.py

```
@@ -223,6 +247,164 @@ def convert_pcm_to_wav(pcm_data: bytes, channels: int = 1, sample_rate: int = 24
     return wav_buffer.read()
 
 
+# =============================================================================
+# Google GenAI Files and Batch API Helper Functions
+# =============================================================================
+
+def create_google_batch_json_content(model: str, num_requests: int = 2) -> str:
```

Ruff (0.14.7)

Unused function argument: `model`

(ARG001)